python/geometry_config.py

```
@@ -138,290 +163,351 @@ def create_config(
-    # interaction point, start of target
-
-    c.target.z0 = 0  # Origin of SHiP coordinate system
-    c.target.z = c.target.z0 + c.target.length / 2.0
-
-    c.hadronAbsorber = AttrDict()
+                slices_material.append(target_yaml_data["M"][i])
+    # Last gap should be 0
+    slices_gap[target_yaml_data["nS"] - 1] = 0
+    print(slices_material, slices_length, slices_gap)
+
+    target_length: float = sum(w + g for w, g in zip(slices_length, slices_gap))
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)